### ISSUE TITLE
Change window title to include possible profile

### ISSUE THREAD

**Issue by Sazu-bit** [SchildiChat/schildichat-desktop#177]
Enhancement request.

I have a range of different profiles set up on my system, but they are near indistinguishable in the system tray or task bar. 

If I hover over them, they'll say something like `SchildiChat | $RoomName`

I would greatly appreciate it if it would say `$ProfileName | SchildiChat | $RoomName`

**Comment by ShadowJonathan** [comment]
I think this can still be relevant

**Comment by SpiritCroc** [comment]
> I think this can still be relevant

Yes, the `longterm` label will exclude it from the stale-automation from striking again :)
